release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,291 @@
+=== eBPF is now the default collection method
+
+In {product-title-short} 3.71, Red Hat has updated the default collection method for Collector to eBPF. Using eBPF for data collection takes advantage of modern kernel features and improves performance over kernel modules by 25%. Additionally, it reduces processed `syscall` events by 99%.
+
+
+[NOTE]
+====
+You should switch to kernel module collection if you are using:
+
+* {ocp} version 3.11, which uses {op-system-base-full} 7. It does not support the features required for performance improvement.
+* SELinux on {op-system-base} 7, which may also prevent the loading of eBPF probes.
+====
+
+[id="deprecation-notices-371"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed, refer to the table below. Additional information about some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+- GA: General Availability
+- TP: Technology Preview
+- DEP: Deprecated
+- REM: Removed
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+|Feature |{product-title-short} 3.69 |{product-title-short} 3.70 | {product-title-short} 3.71
+
+|Ability to add comments to alerts and processes
+|DEP
+|DEP
+|REM
+
+|Anchore, Tenable, and Docker Trusted registry integrations
+|DEP
+|DEP
+|REM
+
+|External authorization plug-in for scoped access control
+|DEP
+|DEP
+|REM
+
+|`FROM` option in the `Disallowed Dockerfile` line policy field
+|GA
+|DEP
+|DEP
+
+| `PodSecurityPolicy` (PSP) Kubernetes objects
+| GA
+| DEP
+| REM
+
+|`RenamePolicyCategory` and `DeletePolicyCategory` API endpoints
+|GA
+|DEP
+|DEP
+
+| `--rhacs` option for the `roxctl` helm output command
+| DEP
+| DEP
+| REM
+
+|`/v1/nodes` and `/v1/images` API endpoint response: `firstNodeOccurrence` response body parameter
+|DEP
+|DEP
+|REM
+
+
+| Permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, `User`, `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, `Risk`, `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, `SignatureIntegration`, `ImageComponent`
+| GA
+| GA
+| DEP
+
+| Retrieving groups by property
+| GA
+| GA
+| DEP
+
+|  `vulns` fields of `storage.Node` object in response payload of `v1/nodes` 
+| GA
+| GA
+| DEP
+
+| `/v1/cves/suppress` and `/v1/cves/unsuppress` 
+| GA
+| GA
+| DEP
+
+
+|====
+
+[id="deprecated-features-371"]
+=== Deprecated features
+
+* Permissions for permission sets will be grouped for simplification. The following list describes the new permissions and indicates the deprecated permissions that will be removed in a future release:
```

Comment:
@kcarmichael08 is it permission sets or Permission sets?

Reply:
I don't think it's specific to ACS, so I don't think capitalized since it's a general term like "permissions."